packages/celotool/src/lib/oracle.ts

```diff
@@ -66,14 +72,19 @@ const oracleAzureContextClusterConfigEnvVars: {
 /**
  * Env vars corresponding to each value for the OracleConfig for a particular context
  */
-const oracleAzureContextOracleConfigEnvVars: {
-  [key in keyof typeof OracleAzureContext]: { [k in keyof OracleConfig]: string }
+const oracleAzureContextOracleIdentityEnvVars: {
+  [key in keyof typeof OracleAzureContext]: {
+    addressAzureKeyVaults: string,
+    addressesFromMnemonicCount: string,
+  }
 } = {
   [OracleAzureContext.PRIMARY]: {
-    identities: envVar.ORACLE_PRIMARY_ADDRESS_AZURE_KEY_VAULTS,
+    addressAzureKeyVaults: envVar.ORACLE_PRIMARY_ADDRESS_AZURE_KEY_VAULTS,
+    addressesFromMnemonicCount: envVar.ORACLE_PRIMARY_ADDRESSES_FROM_MNEMONIC_COUNT,
   },
   [OracleAzureContext.SECONDARY]: {
-    identities: envVar.ORACLE_SECONDARY_ADDRESS_AZURE_KEY_VAULTS,
+    addressAzureKeyVaults: envVar.ORACLE_SECONDARY_ADDRESS_AZURE_KEY_VAULTS,
+    addressesFromMnemonicCount: envVar.ORACLE_SECONDARY_ADDRESSES_FROM_MNEMONIC_COUNT,
```

Comment:
Something we can add later, but it'd be cool to be able to specify an offset for the second mnemonic addresses. Ex. if we have 5 in primary, then we skip the first 5 addresses to prevent overlap when deriving the second 5.

Reply:
Totally agree!